mainapp/templates/mainapp/request_form.html

```diff
@@ -1,75 +1,111 @@
 {% extends 'base.html' %}
 {% load bootstrap3 %}
 {% load static %}
+{% block css %}
+	<style type="text/css">
```

Comment:
fix indent here and everywhere (2spaces seem to be norm for this code base)

Reply:
The original had spaces for html and tabs for javascript and hence the confusion.